stan/math/prim/meta/as_scalar.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef STAN_MATH_PRIM_META_AS_SCALAR_HPP
+#define STAN_MATH_PRIM_META_AS_SCALAR_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <vector>
+#include <stdexcept>
+
+namespace stan {
+namespace math {
+
+/** \ingroup type_trait
```

Comment:
That PR replaced `as_scalar` with now `forward_as` that is more general (not limited to scalars). Since that PR was just merged into develop I guess @rok-cesnovar has not merged changes into this PR yet.

Reply:
You are correct @t4c1, this needs to be updated with the develop of today.

Reply:
This was done. `as_scalar` removed and latest state of meta merged in.